Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,16 @@
 
 import UIKit
 
+public struct MultilineMenuItem: MenuItemType {
```

Comment:
In the case of creating view only menu with MultilineMenuItem, we need to pass [MultilineMenuItem] when initializing the PagingMenuViewController